### PR TITLE
Italian translation update

### PR DESCRIPTION
**Pull request by ppasserini**
Few more Italian strings & fix.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
Please see failing tests

**Comment by ppasserini** [comment]
File fixed

File: resources/lang/it.json [hunk 1]
```diff
@@ -402,5 +404,86 @@
   "Groups": "Gruppi",
   "Performance": "Prestazioni",
   "History": "Cronologia",
-  "Filter Settings": "Impostazioni filtro"
+  "Filter Settings": "Impostazioni filtro",
+  "(Un) Acknowledgement note:": "PP",
+  "(Un)Acknowledgement note:": "PP",
+  "Ack alert": "Ricezione avviso",
+  "Acknowledge Alert": "Recepimento avviso",
+  "Acknowledge until alert clears": "PP",
```
Comment (Jellyfrog) [inline resources/lang/it.json:412]: All of these PP looks wrong?

File: resources/lang/it.json [hunk 2]
```diff
@@ -402,5 +404,86 @@
...
+  "Remove Services for this Service Template": "Rimuovere i servizi per questo modello",
+  "Save notes": "Salva note",
+  "Seconds": "s",
+  "Server Name": "Nome del server",
+  "Service Down": "Servizio down",
+  "Services Templates": "Modelli dei servizi",
+  "The Service Template could not be deleted": "Non è stato possibile eliminare il modello",
+  "The device group could not be deleted": "Non è stato possibile eliminare il gruppo",
+  "To": "A",
+  "Toggle navigation": "Navigazione ????",
```
Comment (Jellyfrog) [inline resources/lang/it.json:480]: .

File: resources/lang/it.json [hunk 3]
```diff
@@ -402,5 +404,86 @@
...
+  "Total Poller Time Per Module": "Tempo totale di poller per modulo",
+  "Port Down": "Porta down",
+  "Port Groups": "Gruppi di porte",
+  "Power Status": "Stato dell'alimentazione",
+  "Priority": "Priorità",
+  "Program": "Programma",
+  "Remote Host": "Host remoto",
+  "Remove Services for this Service Template": "Rimuovere i servizi per questo modello",
+  "Save notes": "Salva note",
+  "Seconds": "s",
```
Comment (Jellyfrog) [inline resources/lang/it.json:473]: This should probably not be shorted to s

**Comment by ppasserini** [comment]
Sorry, too many mistakes. I'll double check all files.

**Pull request closed without merging**